Specs/Scene/ExpressionSpec.js

```diff
@@ -783,6 +922,56 @@ defineSuite([
 
         expression = new Expression('color(\'green\') != color(\'green\')');
         expect(expression.evaluate(frameState, undefined)).toEqual(false);
+
+        expression = new Expression('color(\'green\') !== color(\'green\')');
+        expect(expression.evaluate(frameState, undefined)).toEqual(false);
+    });
+
+    it('evaluates vector operations', function() {
+        var expression = new Expression('+vec3(1, 2, 3)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(new Cartesian4(1, 2, 3, 0));
+
+        expression = new Expression('-vec3(1, 2, 3)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(new Cartesian4(-1, -2, -3, 0));
+
+        expression = new Expression('vec2(1, 2) + vec4(3, 4, 5, 6)');
```

Comment:
The vector types are needed to support vector properties in the batch table binary, which can be vec2, vec3, vec4, or accessing point positions with `${POSITIONS}` as well as math operations. Having vec4 as the base type also makes it easier to interact with colors (should colors and vectors mix?).

In line with other vector libraries I can support vec2, vec3, and vec4 as distinct types - however it will add more complexity.